pkgs/applications/radio/gnuradio/wrapper.nix

```diff
@@ -84,9 +84,7 @@ let
         at-spi2-core
       ]}"
     ]
-    ++ lib.optionals (extraPackages != []) [
-      "--prefix" "GRC_BLOCKS_PATH" ":" "${lib.makeSearchPath "share/gnuradio/grc/blocks" extraPackages}"
-    ]
+    ++ [ "--prefix" "GRC_BLOCKS_PATH" ":" "${lib.makeSearchPath "share/gnuradio/grc/blocks" (extraPackages ++ [ unwrapped ])}" ]
```

Comment:
Thanks for this, makes sense!

Reply:
Quoting myself:

> Thanks for this, makes sense!

However, I'm surprised GnuRadio doesn't make that path part of the search path by default... Are you sure this is needed?

Reply:
Hm, I was a bit surprised, too, since it worked before without it. But when I launch `gnuradio-companion` from a `nix-shell -p gnuradio` I get the error message above... Might of course be some impurity in my environment.